submodules/faxbox/views/edit.html

```diff
@@ -123,11 +123,11 @@ <h3>{{ i18n.callflows.faxbox.notification_emails }}</h3>
 					</div>
 				</div>
 
-			{{#if faxbox._read_only.custom_smtp_address}}
+			{{#if faxbox.custom_smtp_address}}
 				<div class="clearfix">
 					<label for="custom_smtp_address">{{ i18n.callflows.faxbox.smtp_email_address }}</label>
```

Comment:
Is this filed the `custom_smtp_address`, or the `smtp_email_address`? I think they are interchanged with the field above.

I think there is also a confusion in the ticket, because according to the UI image the value should be displayed in the input field above:
![image](https://github.com/2600hz/monster-ui-callflows/assets/893877/8dbeec43-d12a-4754-8096-9857b017f520)

So basically the question is, what should be editable? The SMTP address or the SMTP e-mail address?